hack/lib/util.sh

```diff
@@ -457,7 +457,7 @@ kube::util::godep_restored() {
 kube::util::ensure_clean_working_dir() {
   if ! git diff --exit-code; then
     echo -e "\nUnexpected dirty working directory."
-    exit 1
+    return 1
```

Comment:
Consistency would be best, so then we know what to expect either way.. If we go with `return` here, should we scrub the rest of `utils.sh` to change it over to use `return` as well? (Same question if we go with the `exit` explicit option.)